static/schemas/2.0/skus.2.0.json

```diff
@@ -0,0 +1,248 @@
+{
+    "title": "Skus.2.0",
+    "definitions": {
+        "SkusRules": {
+            "description": "Possible Rules a Sku can use",
+            "type": "array",
+            "items": {
+                "type": "object",
```

Comment:
@BillyAbildgaard I think it makes sense to enforce what's in the documentation with the schema, rather than the full list (since a lot of those validations aren't really applicable to this task anyways).